src/OCServo.cpp

```diff
@@ -0,0 +1,481 @@
+#include "OCServo.h"
+
+/* CONSTRUCTORS */
+OCServo::OCServo(byte id, SoftwareSerial *serialPort) : id(id), serial(serialPort){
+    this->portType = SERIAL_SOFTWARE;
+}
+
+OCServo::OCServo(byte id, HardwareSerial *serialPort) : id(id), serial(serialPort){
+    this->portType = SERIAL_HARDWARE;
+}
+
+/* PRIVATE METHODS */
+byte OCServo::getChecksum(byte *data, int size) {
+    byte checksum = 0;
+    for (byte i = 2; i < size; i++)
+    {
+        checksum += data[i];
+    }
+    return ~(checksum & 0xFF);
+}
+
+byte OCServo::baudRateToByte(long baudrate) {
+    byte value = 0x00;
+
+    switch (baudrate) {
+        case 500000:
+            value = OCS_BAUDRATE_500K;
+            break;
+        case 250000:
+            value = OCS_BAUDRATE_250K;
+            break;
+        case 128000:
+            value = OCS_BAUDRATE_128K;
+            break;
+        case 115200:
+            value = OCS_BAUDRATE_115200;
+            break;
+        case 76800:
+            value = OCS_BAUDRATE_76800;
+            break;
+        case 57600:
+            value = OCS_BAUDRATE_57600;
+            break;
+        case 38400:
+            value = OCS_BAUDRATE_38400;
+            break;
+        default:
+            // Fallthrough to default 1M
+        case 1000000:
+            value = OCS_BAUDRATE_1M;
+            break;
+    }
```

Comment:
Minor: small simplification

```suggestion
uint8_t OCServo::baudRateToByte(uint32_t baudrate) const {
    switch (baudrate) {
        case 500000:   return OCS_BAUDRATE_500K;
        case 250000:   return OCS_BAUDRATE_250K;
        case 128000:   return OCS_BAUDRATE_128K;
        case 115200:   return OCS_BAUDRATE_115200;
        case 76800:    return OCS_BAUDRATE_76800;
        case 57600:    return OCS_BAUDRATE_57600;
        case 38400:    return OCS_BAUDRATE_38400;
        default:
        case 1000000:  return OCS_BAUDRATE_1M;
    }
}
```